src/fable/Fable.Client.Suave/Main.fs

```diff
@@ -188,14 +259,17 @@ open Suave.Successful
 let main argv =
     let projOpts = getProjectOptions()
     let checker = FSharpChecker.Create(keepAssemblyContents=true)
-
+    let port = match argv with [| port |] -> int port | _ -> 8083
+     
     let setCORSHeaders =
         setHeader  "Access-Control-Allow-Origin" "*"
         >=> setHeader "Access-Control-Allow-Headers" "content-type"
 
     let handle (req : HttpRequest) = 
         let getString rawForm = System.Text.Encoding.UTF8.GetString(rawForm)
-        req.rawForm |> getString |> makeCompiler |> compile checker projOpts |> OK
+        let code = req.rawForm |> getString
+        let result = compile checker code projOpts (makeCompiler())        
```

Comment:
Recently I was thinking that maybe the reason the previous Try Fable site failed was because the compiler was trying to satisfy several requests in parallel. Beforehand it was a more serious problem because I was writing to disk with every request. Now the Suave client uses the virtual file system but even with that we may need to protect `compile` calls against parallelisation (I'm not a Suave expert so I'm not sure if this is needed at all) with a `lock` or a `MailboxProcessor`.

Besides that, there are some static values we may have to reset, like the store for inline functions (I know, I know, that's a very bad functional programming practice :wink:): https://github.com/fable-compiler/Fable/blob/master/src/fable/Fable.Compiler/FSharp2Fable.fs#L836-L838